core/src/io/anuke/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -258,6 +260,15 @@ public void drawPlace(int x, int y, int rotation, boolean valid){
         Draw.reset();
     }
 
+    @Override
+    public void draw(Tile tile) {
+        super.draw(tile);
+        if(tile.entity.power.graph.getPowerNodeColor() == null || tile.block() instanceof PowerSource) return;
+        Draw.color(tile.entity.power.graph.getPowerNodeColor());
+        Draw.rect(led, tile.drawx(), tile.drawy(), (id%4)*90);
```

Comment:
What is `(id%4)*90` for?

Reply:
i don't know, the code where i copied that from had it 😂 (cracks iirc)